apps/src/templates/CollapserIcon.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Radium from 'radium';
+
+const collapseIconStyles = {
```

Comment:
NIT: in general our code is pretty consistent about calling this object `styles`, and since its local to this module the extra namespacing doesn't really add much.

Reply:
Oh I did this when I thought I was going to name the prop `styles` and didn't want a naming conflict, but I renamed to `style`. Will update!